pyproject.toml

```diff
@@ -77,18 +77,19 @@ torch = {version = ">=1.6.0, <3.0.0", optional = true}
 pytorch-lightning = {version = ">=1.6.0, <3.0.0", optional = true}
 
 ipywidgets = {version = ">=7.7,<8.2", optional = true}
+plotly = "^5.22.0"
```

Comment:
Please mark it optional since it's in extras (and run `poetry lock` after it)

Reply:
Done